programming_examples/basic/vector_scalar_mul/Makefile

```diff
@@ -43,7 +43,7 @@ build/final_trace_${data_size}.xclbin: build/aie_trace.mlir build/scale.o
 	cd ${@D} && aiecc.py --aie-generate-cdo --no-compile-host --xclbin-name=${@F} \
 				--aie-generate-ipu --ipu-insts-name=insts_${data_size}.txt $(<:%=../%)
 
-${targetname}_${data_size}.exe: test.cpp
+${targetname}.exe: test.cpp
```

Comment:
We want to keep the ${data_size} in the exe name to avoid deadlock when people change the data_size but did not do a make clean

Reply:
Reverted back to this in a later commit in this PR